boards/arc/arduino_101_sss/arduino_101_sss.dts

```diff
@@ -66,4 +66,13 @@
 
 &spi1 {
 	status = "ok";
+
+	bmi160@0 {
```

Comment:
This sensor is on the board, not a shield?

Should the slave number be 1 instead of 0? (it was set to 1 in `samples/sensor/bmi160/prj.conf`)

Reply:
Should have been 1.  Fixed now.